CHANGELOG.md

```diff
@@ -18,10 +18,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Removed
 - none yet
 
+
+## [v0.4.2 - 2017-11-14]
```

Comment:
Heresy! You have put the date between the brackets. For other versions, it is just the version number, not the date, that is between the brackets.

Reply:
😱 

Thanks for spotting this !